src/components/Events/EventForm.vue

```diff
@@ -186,6 +205,7 @@ export default {
           start: this.eventProp.details.start,
           end: this.eventProp.details.end,
         },
+        price: ConversionUtils.centsToDollars(this.eventProp.price),
```

Comment:
What is eventProp? I'm confused why we convert to dollars here and back to cents on line 280